src/Game/UI/Gumps/WorldMapGump.cs

```diff
@@ -86,6 +121,13 @@ public override void Restore(XmlElement xml)
             if (int.TryParse(xml.GetAttribute("zoomindex"), out int value))
                 _zoomIndex = (value >= 0 && value < _zooms.Length) ? value : 4;
 
+            _showCoordinates = bool.Parse(xml.GetAttribute("showcoordinates"));
```

Comment:
bool.TryParse instead of bool.Parse 
or it will throw errors

Reply:
Ok, bool.Parse is used throughout the class -- you want all of them refactored?

Reply:
Yes just to avoid exceptions with retro compatibility

Reply:
Made those changes to the bool parsing.